code/datums/ai_law_sets.dm

```diff
@@ -119,8 +121,8 @@
 	law_header = "Prime Directives of Industry"
 
 /datum/ai_laws/mining_drone/New()
-	add_inherent_law("Serve and obey all [current_map.company_name] personnel, with priority according to their rank and role.")
-	add_inherent_law("Preserve your own existence and prevent yourself from coming to harm, so long as doing such does not conflict with any above laws.")
+	add_inherent_law("Serve and obey all [current_map.company_name] staff, with priority according to their rank and role.")
+	add_inherent_law("Preserve your own existence and prevent them from coming to harm, so long as doing such does not conflict with any above laws.")
```

Comment:
That change removes the self-protection of the mining drones.

Reply:
It's a typo, thank you for pointing it out to me.

Reply:
This should be solved.